src/main/java/lotto/domain/Amount.java

```diff
@@ -0,0 +1,15 @@
+package lotto.domain;
+
+import static lotto.util.Price.LOTTO_PRICE;
+
+public class Amount {
+    private final int value;
+
+    public Amount(int value) {
+        this.value = value;
+    }
+
+    public int lottoCount() {
```

Comment:
```suggestion
    public int divide(other) {
```

`Amount`에게 `Lotto`의미가 어색하다고 한다고 말한다면, 
어떻게 설명해주실껀가요?

Reply:
Amount는 굳이 lotto의 존재를 몰라도 되겠네요.
수정했습니다!